### ISSUE TITLE
Improve consistency in working with people and organizations

### ISSUE THREAD

**Issue by dtarb** [hydroshare/hydroshare#1399]
This suggestion is motivated by #1221 where there is work on a model for people and organizations, by suggestions from @ChristinaB for automatically selecting authors/contributors in resource creation from from other authors/contributors in the system who may not be HydroShare users and from the concepts in the original party model (a brief description of which is in http://academicworks.cuny.edu/cc_conf_hic/314/ figure 2).

I think we need a consistent model for how people and organizations (authors, contributors, users) are represented.  This should not be limited to users, and should be consistent with the model being used for time series #1221 that arose out of much ODM2 effort.  A physical implementation of such a model  would then enable enhanced author/contributor content entry in a consistent way.

**Comment by pkdash** [comment]
@dtarb Is this still an issue?

**Comment by aphelionz** [comment]
@pkdash @dtarb What is the status of this?

**Comment by pkdash** [comment]
@aphelionz No work has been done to resolve this issue.

**Comment by horsburgh** [comment]
@dtarb - I suggest that we close this issue from 2016. Perhaps we could work on a consistent implementation of a party model, but we have working implementations for adding people and organizations to resources in the context of owners, authors, and contributors. This issue doesn't explain what actual problems exist or how to fix them.

I'm also confused by the link to issue #1221 - that's a long dialog about time series implementation that is hard to parse for any useful information about how people and organizations are implemented.

**Comment by dtarb** [comment]
Agreed.  I find myself also not clear on the #1221 link.  It may have been related to ODM2 implementation of a formal representation for metadata, but if work is to be done there needs to be an impetus that there is not at present.
